javalib/src/main/scala/java/lang/Runtime.scala

```diff
@@ -18,6 +19,17 @@ class Runtime private () {
   lazy val setupAtExitHandler = {
     stdlib.atexit(() => Runtime.getRuntime().runHooks())
   }
+
+  lazy val setupSignalHandler = {
+    signal.signal(signal.SIGINT, handleSignal(_))
+    signal.signal(signal.SIGTERM, handleSignal(_))
+  }
+
+  private def handleSignal(sig: CInt): Unit = {
+    Runtime.getRuntime().runHooks()
```

Comment:
I think it won't work when we'll have multiple threads running. Starting `runHooks` would also start `scala.scanative.runtime.JoinNonDaemonThreads` which would wait for all thread to finish. We should at least remove this shutdown hook. 
Also signal handler might be run by any of the threads. Which might affect its execution.